drivers/serial/uart_nrfx_uarte.c

```diff
@@ -502,42 +502,45 @@ static int uarte_instance_init(struct device *dev,
 	return 0;
 }
 
-#define UART_NRF_UARTE_DEVICE(idx)					  \
-	DEVICE_DECLARE(uart_nrfx_uarte##idx);				  \
-	UARTE_##idx##_CREATE_TX_BUFF;					  \
-	static struct uarte_nrfx_data uarte_##idx##_data = {		  \
-		UARTE_##idx##_DATA_INIT					  \
-	};								  \
-	static const struct uarte_nrfx_config uarte_##idx##_config = {	  \
-		.uarte_regs = (NRF_UARTE_Type *)DT_UART_##idx##_BASE, \
-		UARTE_##idx##_CONFIG_INIT				  \
-	};								  \
-	static int uarte_##idx##_init(struct device *dev)		  \
-	{								  \
-		const struct uarte_init_config init_config = {		  \
-			.pseltxd = DT_UART_##idx##_TX_PIN,	  \
-			.pselrxd = DT_UART_##idx##_RX_PIN,	  \
-			UARTE_##idx##_NRF_HWFC_CONFIG			  \
-			.parity = UARTE_##idx##_NRF_PARITY_BIT,		  \
-			.baudrate = DT_UART_##idx##_BAUD_RATE	  \
-		};							  \
-		UARTE_##idx##_INTERRUPTS_INIT();			  \
-		return uarte_instance_init(dev,				  \
-				&init_config,				  \
-				UARTE_##idx##_INTERRUPT_DRIVEN);	  \
-	}								  \
-	DEVICE_AND_API_INIT(uart_nrfx_uarte##idx,			  \
-			DT_UART_##idx##_NAME,			  \
-			uarte_##idx##_init,				  \
-			&uarte_##idx##_data,				  \
-			&uarte_##idx##_config,				  \
-			PRE_KERNEL_1,					  \
-			CONFIG_KERNEL_INIT_PRIORITY_DEVICE,		  \
+#define UART_NRF_UARTE_DEVICE(idx)					       \
+	DEVICE_DECLARE(uart_nrfx_uarte##idx);				       \
+	UARTE_##idx##_CREATE_TX_BUFF;					       \
+	static struct uarte_nrfx_data uarte_##idx##_data = {		       \
+		UARTE_##idx##_DATA_INIT					       \
+	};								       \
+	static const struct uarte_nrfx_config uarte_##idx##_config = {	       \
+		.uarte_regs = (NRF_UARTE_Type *)			       \
+			DT_NORDIC_NRF_UARTE_UART_##idx##_BASE_ADDRESS,	       \
+		UARTE_##idx##_CONFIG_INIT				       \
+	};								       \
+	static int uarte_##idx##_init(struct device *dev)		       \
+	{								       \
+		const struct uarte_init_config init_config = {		       \
+			.pseltxd = DT_NORDIC_NRF_UARTE_UART_##idx##_TX_PIN,     \
+			.pselrxd = DT_NORDIC_NRF_UARTE_UART_##idx##_RX_PIN,     \
```

Comment:
All these `\` characters in the macro should be aligned (see also line 524). Currently these three lines exceed the 80 characters length limit.